### ISSUE TITLE
render events with duration longer than 1 hour

### ISSUE THREAD

**Issue by manuargue** [rouftom/react-mui-scheduler#5]
Hi, I've used the sample code (see [here](https://codesandbox.io/s/react-mui-scheduler-forked-rrm7md?file=/src/App.js)) with a single event that spans 2 hours, from 4:00 to 6:00 AM as follows:
```js
var events = [
    {
      id: "event-1",
      label: "Medical",
      startHour: "04:00 AM",
      endHour: "06:00 AM",
      date: "2022-04-01",
      // more fields...
    }
  ];
```
But it seems that the event is always rendered to span 1 hour:
![image](https://user-images.githubusercontent.com/511857/166153505-782c4cc5-107e-4c5d-b24f-446669834729.png)

Is there any option am I missing to configure? Thanks

**Comment by rouftom** [comment]
No, you haven't missed any configuration. Unfortunately due to lack of time, this functionality has not yet been developed but it will be soon.

**Comment by vicgonzr** [comment]
Hello, someone was able to solve render events with duration longer than 1 hour
